README.md

```diff
@@ -1,24 +1,116 @@
-application up and running.
+  <h3><b>Blog app</b></h3>
 
-Things you may want to cover:
+</div>
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Configuration
+# 📖 Blog app<a name="about-project"></a>.
 
-* Database creation
+The Blog app will be a classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/es/">Ruby</a></li>
+    <li><a href="https://github.com/github/rubocop-github/blob/main/STYLEGUIDE.md">Ruby styles guide</a></li>
+    <li><a href="https://www.ruby-lang.org/es/">Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **Use Gitflow**
+- **Implement classes and objects**
+- **Use encapsulation and inheritance**
+- **Ruby styles reviewed with Rubocop Linters**
+- **Use composition**
+- **Ruby on rails**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This repository includes files with ruby lenguage.
+
+### Prerequisites
+
+In order to run this project you need to have Ruby installed, check [this](https://www.ruby-lang.org/en/) documentation in order to know how to install it.
+-Install Rails
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-folder https://github.com/francksefu/blog_web.git
+  git clone 
+
+### Usage
+
+Comming soon ...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Franck Sefu**
+
+- GitHub: [francksefu](https://github.com/francksefu)
+- Twitter: [francksefu](https://twitter.com/franck_sefu)
+- LinkedIn: [francksefu](https://www.linkedin.com/in/franck-sefu/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Create table by using rails commands**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/francksefu/blog_web.git/issues).
```

Comment:
Great job on updating the link. You worked on the comment. 👍 However, it seems that the link to the issue page is still broken. To ensure that users can be redirected to the proper issue page, I will recommend you to update the link. Keep up the good work! 👍📜
![image](https://github.com/francksefu/blog_web/assets/55840999/a408af56-fa27-42c3-b5a7-1abaad1bd139)